gx_glue_catalog_tutorial/infra/aws_data.tf

```diff
@@ -0,0 +1,5 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+data "aws_iam_policy" "AWSGlueServiceRole" {
+  name = "AWSGlueServiceRole"
+}
```

Comment:
Can you add newlines to all of the .tf files?

Reply:
We could enforce this with a `pre-commit` hook.
https://pre-commit.com/#quick-start
and the `pre-commit` minimal CI
https://pre-commit.ci/

I'll submit a PR with a minimal pre-commit configuration.